scripts/generate_topographic_data.py

```diff
@@ -0,0 +1,87 @@
+"""
+Script that takes the SRTM 30m worldwide elevation maps and creates a
+single representation of the area in NetCDF and GeoTIFF formats
+
+The SRTM files can be downloaded from NASA, and are in CRS EPSG:4326
+"""
+import os.path
+import glob
+import rasterio
+from rasterio.merge import merge
+from rasterio.warp import Resampling
+from nowcasting_dataset.geospatial import OSGB
+import rioxarray
+
+dst_crs = OSGB
+
+# Go through, open all the files, combined by coords, then save out to NetCDF, or GeoTIFF
+files = glob.glob("/SRTM/*.tif")
+out_dir = "/SRTM_TEMP/"
+
+
+upscale_factor = 0.12  # 30m to 250m-ish, just making it small enough files to actually merge
+for f in files:
+    with rasterio.open(f) as dataset:
+
+        # resample data to target shape
+        data = dataset.read(
+            out_shape=(
+                dataset.count,
+                int(dataset.height * upscale_factor),
+                int(dataset.width * upscale_factor),
+            ),
+            resampling=Resampling.bilinear,
+        )
+
+        # scale image transform
+        transform = dataset.transform * dataset.transform.scale(
+            (dataset.width / data.shape[-1]), (dataset.height / data.shape[-2])
+        )
+        name = f.split("/")[-1]
+        # Set the nodata values to 0, as nearly all ocean.
```

Comment:
Perhaps in a later PR, it might be nice to _also_ give the ML model a binary map of "ocean versus land".